contracts/NFTStorage.sol

```diff
@@ -537,7 +537,7 @@ contract NFTStorage is IERC721ReceiverUpgradeable, Initializable, AccessControlU
     }
 
     // Takes bridgedNFT not meta to avoid stack too deep
-    function _withdrawWeaponFromBridge(uint256 bridgedNFT, uint256 chainId, uint256 sourceId) internal returns (uint256 mintedId) {
+    function _withdrawWeaponFromBridge(uint256 bridgedNFT, uint256 /*chainId*/, uint256 /*sourceId*/) internal returns (uint256 mintedId) {
```

Comment:
Why would we keep these params at all? @bilbolPrime any objections to removing them?

Reply:
You could make an argument that the values might be used in the future and thus are a good future flexibility thing.

Also, changing them now means revving the bot and making sure it's the transition is carefully rolled out across all chains.  (Or make the bot smart enough to detect which flavor of the method the contract has.)

Reply:
I am good with this change and it's seamless for the bot. Let's not do any migrations for this contract though as it's scheduled for a major upgrade (https://github.com/CryptoBlades/cryptoblades/pull/1143).